lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -159,34 +169,18 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     Tensor *w = getTensorByName(op.input(1));
 
     // Transpose the weights to the right format. Glow expects to read the
-    // weights in the format CRSK. Caffe2 stores the operators as KCRS.
-    // C - output_depth, R - filter_height, S - filter_width, K - input_depth.
+    // weights in the format NHWC.
     Tensor wtag;
-    w->transpose(&wtag, NCHW2NHWC);
+    if (typeName != "Conv" && order == "NHWC")
+      wtag.assign(w);
+    else
```

Comment:
Can you explain in a comment what's going on here? Also, please add braces around the statements.